firmware/lucidgloves-firmware/MinMaxCalibration.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+template<typename T>
+struct MinMaxCalibration {
+ public:
+  MinMaxCalibration(T output_min_,T output_max_, bool clamp_) :
+    output_min(output_min_),
+    output_max(output_max_),
+    value_min((output_min_ + output_max_) / 2.0f),
```

Comment:
This limits the range of calibration to around the center values. Max can never be less than max/2, which should actually be allowed.